Assets/PatchKit Patcher/Scripts/AppData/FileSystem/FileOperations.cs

```diff
@@ -159,5 +159,10 @@ private static void MoveInternal(string sourceFilePath, string destinationFilePa
                 throw;
             }
         }
+
+        public static long GetSizeFile(string filePath)
+        {
+            return !string.IsNullOrEmpty(filePath) && File.Exists(filePath) ? new FileInfo(filePath).Length : 0;
```

Comment:
Between `File.Exists()` and `new FileInfo()`, a file can be deleted.

FileInfo contains Exist property, which should be used to check if the entry exists:
https://docs.microsoft.com/pl-pl/dotnet/api/system.io.fileinfo.exists?view=net-6.0#system-io-fileinfo-exists

Then use the same instance to get its Length.

Reply:
Done